### ISSUE TITLE
Technical Documentation

### ISSUE THREAD

**Issue by RobertBornemann** [NinaDaPraia/Nina#6]
- As a new developer
- I want to read the technical documentation of Nina
- So I can integrate myself quickly and contribute to the project

Checklist:  
Introdução do guia de estilo 
Guia de como preparar ambientes 
Descrição breve do domínio do escopo 
Dependências necessárias 
Tech Stack usado

**Comment by RobertBornemann** [comment]
So how are we going to handle the API documentation with Swagger? Can we incorporate the elements of documentation in there as well?

**Comment by sephioh** [comment]
No. Swagger must a yaml file to generate this its docs. We should focus on: 
1) improve the current onboarding experience;
2) list important decisions that we should all be aware of it;
3) list our communication channels;

I will put this on, but i will need some feedback about it, so you all fell free to ask what you want about this.
